dbclient/WorkspaceClient.py

```diff
@@ -247,7 +248,7 @@ def download_notebooks(self, ws_log_file='user_workspace.log', ws_dir='artifacts
             for notebook_data in fp:
                 notebook_path = json.loads(notebook_data).get('path', None).rstrip('\n')
                 dl_resp = self.download_notebook_helper(notebook_path, export_dir=self.get_export_dir() + ws_dir)
-                if 'error_code' not in dl_resp:
+                if 'error' not in dl_resp:
```

Comment:
If I recall, the response of `dl_resp` is a json object, so `error_code` is a key check for this json object. Changing to `error` may not produce the same result. Have we tested this error code and return path?

Reply:
dl_resp is returned from `download_notebook_helper` which is `error` as key in the json object. 
```
with open(self.get_export_dir() + 'failed_notebooks.log', 'a') as err_log:
            if resp.get('error', None):
                err_msg = {'error': resp.get('error'), 'path': notebook_path}
                err_log.write(json.dumps(err_msg) + '\n')
                return err_msg
```

Reply:
can we double check on this? How has this been working if `download_notebook_helper` indeed returns `error` as the key?

Reply:
doubled checked the log file - `failed_notebooks.log`. The format is
```
{"error": "DatabricksServiceException: RESOURCE_DOES_NOT_EXIST: Path (/Users/ekl-backup-1/Intro Notebooks / Intro R Notebooks) doesn't exist.", "path": "/Users/ekl-backup-1/Intro Notebooks / Intro R Notebooks"}
```
I think it wasn't working before

Reply:
got it